### PR TITLE
Renames loyalty implants to mindshield implants

### PR DESCRIPTION
**Pull request by imtakingabreakdontatme**
It has been a continuous source of confusion to players that loyalty implants do not, in fact, make you loyal. They only protect against brainwashing/conversion antags.

Hopefully this helps clarify somewhat that it will not at all work on traitors, changelings, or anyone else who is a badguy under their own free will.

Open to changing the fluff strings/implant name but it should really be something more clear than "loyalty"

### CODE REVIEW

**Comment by Robustin** [comment]
Sure why not. Ultimately people are idiots though, "but I thought traitors were all brainwashed?!?!"

File: code/game/gamemodes/handofgod/_handofgod.dm [hunk 1]
```diff
@@ -183,7 +183,7 @@ var/global/list/global_handofgod_structuretypes = list()
 /datum/game_mode/proc/add_hog_follower(datum/mind/follower_mind, colour = "No Colour")
 	var/mob/living/carbon/human/H = follower_mind.current
 	if(isloyal(H))
-		H << "<span class='danger'>Your loyalty implant blocked the influence of the [colour] deity. </span>"
+		H << "<span class='danger'>Your miendshield implant blocked the influence of the [colour] deity. </span>"
```
Comment (CosmicScientist) [inline code/game/gamemodes/handofgod/_handofgod.dm]: mindshield

File: code/game/gamemodes/shadowling/shadowling_abilities.dm [hunk 2]
```diff
@@ -226,11 +226,11 @@
...
 					sleep(20)
 					if(isloyal(target))
-						usr << "<span class='notice'>They are enslaved by Nanotrasen. You begin to shut down the nanobot implant - this will take some time.</span>"
+						usr << "<span class='notice'>Their mind is protected by an implant - you begin to shut it down - this will take some time.</span>"
 						usr.visible_message("<span class='warning'>[usr] pauses, then dips their head in concentration!</span>")
-						target << "<span class='boldannounce'>You feel your loyalties begin to weaken!</span>"
+						target << "<span class='boldannounce'>You feel your resolve begin to weaken!</span>"
 						sleep(100) //10 seconds - not spawn() so the enthralling takes longer
-						usr << "<span class='notice'>The nanobots composing the loyalty implant have been rendered inert. Now to continue.</span>"
+						usr << "<span class='notice'>The mindshield implant have been rendered inert. Now to continue.</span>"
```
Comment (CosmicScientist) [inline code/game/gamemodes/shadowling/shadowling_abilities.dm]: has been

File: code/game/objects/items/weapons/implants/implant_loyality.dm [hunk 3]
```diff
@@ -11,7 +11,7 @@
 				<b>Important Notes:</b> Personnel injected with this device tend to be much more loyal to the company.<BR>
 				<HR>
 				<b>Implant Details:</b><BR>
-				<b>Function:</b> Contains a small pod of nanobots that manipulate the host's mental functions.<BR>
+				<b>Function:</b> Contains a small pod of nanobots that will protect the hosts mental functions from outside influences.<BR>
```
Comment (CosmicScientist) [inline code/game/objects/items/weapons/implants/implant_loyality.dm]: Just curious, why not host's?

File: code/game/objects/items/weapons/implants/implant_loyality.dm [hunk 4]
```diff
@@ -20,13 +20,13 @@
 /obj/item/weapon/implant/loyalty/implant(mob/target)
 	if(..())
 		if((target.mind in (ticker.mode.head_revolutionaries | ticker.mode.get_gang_bosses())) || is_shadow_or_thrall(target))
-			target.visible_message("<span class='warning'>[target] seems to resist the implant!</span>", "<span class='warning'>You feel the corporate tendrils of Nanotrasen try to invade your mind!</span>")
+			target.visible_message("<span class='warning'>The implant beeps out a warning. [target] is too corrupt to save!</span>", "<span class='warning'>You feel something trying to heal your mind...but you're too far gone.</span>")
```
Comment (CosmicScientist) [inline code/game/objects/items/weapons/implants/implant_loyality.dm]: mind... but

File: code/game/objects/items/weapons/implants/implant_loyality.dm [hunk 5]
```diff
@@ -35,22 +35,22 @@
 		if(target.mind in ticker.mode.red_deity_followers|ticker.mode.blue_deity_followers)
 			ticker.mode.remove_hog_follower(target.mind)
 		if((target.mind in ticker.mode.cult) || (target.mind in ticker.mode.blue_deity_prophets|ticker.mode.red_deity_prophets))
-			target << "<span class='warning'>You feel the corporate tendrils of Nanotrasen try to invade your mind!</span>"
+			target << "<span class='warning'>You feel something trying to heal your mind...but you're too far gone!</span>"
```
Comment (CosmicScientist) [inline code/game/objects/items/weapons/implants/implant_loyality.dm]: mind... but

File: code/game/objects/items/weapons/implants/implant_loyality.dm [hunk 6]
```diff
@@ -59,8 +59,8 @@
 
 
 /obj/item/weapon/implantcase/loyalty
-	name = "implant case - 'Loyalty'"
-	desc = "A glass case containing a loyalty implant."
+	name = "implant case - 'mindshield'"
+	desc = "A glass case containing a mindshield implant."
```
Comment (CosmicScientist) [inline code/game/objects/items/weapons/implants/implant_loyality.dm]: Just curious, why not capitalise it to be Mindshield for name like the old one was?

**Comment by CosmicScientist** [comment]
No reason to rename the actual class names to mindshield?

**Comment by Steelpoint** [comment]
While I think Loyalty Implants are a nice name, I do think that this will help clearing any confusion from new players or idiot's who don't know anything sans aiming the pointy end of the stick towards the officer.

**Comment by Cheridan** [comment]
>the single game mechanic in ss13 that some people like to roleplay out
>lets remove it

i don't even

**Comment by Robustin** [comment]
Yea I'll trade in the occasional "HAIL NANOTRASEN" to end the "WHY IS THE PROMOTED OFFICER KILLING EVERYONE WE GAVE HIM AN IMPLANT?!?" crap.

**Comment by CosmicScientist** [comment]
@Robustin don't worry, I'll force the user to say "Damn Nanotrasen suppressing my freedom of thought!"

**Comment by KorPhaeron** [comment]
@Cheridan

A dozen people screaming in deadchat isn't roleplay its confusion and salt

File: code/game/objects/items/weapons/implants/implant_loyality.dm (hunk 3 above)
Comment (PKPenguin321) [inline code/game/objects/items/weapons/implants/implant_loyality.dm]: pretty sure the correct form would be "host's" as it's a possessive. "hosts" is a plural which would refer to multiple hosts, which is not the goal of this sentence. :cop: grammar police :cop:

**Comment by PKPenguin321** [comment]
i dont care whether or not this gets merged, i'll still roleplay as a brainwashed corporate nazi whenever i get implanted

File: code/game/objects/items/weapons/implants/implant_loyality.dm (hunk 3 above)
Comment (CosmicScientist) [inline code/game/objects/items/weapons/implants/implant_loyality.dm]: Alright yeah you're right. Now add to my other line comments to force Kor to fix his shizz through obnoxious peer pressure!

**Comment by Shapsy** [comment]
The thought of someone acting super brainwashed and loyal to the death to Nanotrasen purely out of placebo effect is more than hilarious enough to make this worth it

**Comment by PKPenguin321** [comment]
no no its the death to those that oppose nanotrasen, get it right communist scum

**Comment by CosmicScientist** [comment]
So you aren't changing the actual names to path/path/mindshield for maintainability?

**Comment by lordpidey** [comment]
Drat.  I used to love saying "I used to think mind control nanobots were unethical, but I now trust Nanotransen to use the technology responsibly."

**Comment by KorPhaeron** [comment]
Well nobody has said yes/no other than Cheridan on this so I'm not gonna maintain it

**Pull request closed without merging**